### ISSUE TITLE
XFCE PPC64 tarball comes with GNOME too

### ISSUE THREAD

**Issue by EchedelleLR** [AOSC-Dev/aosc-os#12]
I don't understand the situation nor its explained in the wiki.

**Comment by MingcongBai** [comment]
@EchedeyLR It is expected that some GNOME components may be installed with XFCE (and even KDE) distributions since they provide some  necessary functionalities (GNOME Desktop API used by NetworkManager Applets, for instance)...

**Comment by EchedelleLR** [comment]
The whole desktop? gnome-desktop package is included

**Comment by MingcongBai** [comment]
@EchedeyLR That’s just an unfortunate package name (upstream’s decision), gnome-desktop contains not the desktop itself but the shared API used by its (and numerous other FreeDesktop.org) components.

**Comment by EchedelleLR** [comment]
gnome-online-accounts I guess its the same as other distributions. Which is the purpose?

**Comment by MingcongBai** [comment]
@EchedeyLR It’s also a library of APIs used by some GNOME applications (Cheese comes to mind) to provide online account integration and management.

Just to explain the situation at hand - our distribution don’t normally split packages and virtually all packages are shipped as they are provided by the upstream. This takes up more HDD space, but we consider this a measure to simplify deployment and management (and frankly also for our own sake).

**Comment by EchedelleLR** [comment]
I have no issues with that, only I got confused reading similar names and expecting other thing. At least the situation is understood and without the issues I thought. I checked that xfce also came with cheese and evince as you referenced the first.

Checking packages.aosc.io also helped me to understand. I am glad to read that gnome-online-accounts doesnt point nor provides evolution-data-server and other non-needed things.
